Gems/PhysX/Code/Source/FixedJointComponent.cpp

```diff
@@ -91,5 +92,81 @@ namespace PhysX
                 leadFollowerInfo.m_followerBody->m_bodyHandle);
             m_jointSceneOwner = leadFollowerInfo.m_followerBody->m_sceneOwner;
         }
+        m_nativeJoint = FixedJointComponent::GetNativeJoint();
+        if (m_nativeJoint)
+        {
+            JointRequestBus::Handler::BusConnect(AZ::EntityComponentIdPair(GetEntityId(), GetId()));
+        }
+    }
+
+    void FixedJointComponent::DeinitNativeJoint()
+    {
+        JointRequestBus::Handler::BusDisconnect();
+        m_nativeJoint = nullptr;
     }
+
+    physx::PxFixedJoint* FixedJointComponent::GetNativeJoint() const
```

Comment:
update: Please update this to return a `const` pointer or remove the `const` (or provide both const and non-const overloads)

Reply:
It's a private function to obtain the native joint, it's not really a getter. It cannot return const because the result is setting it to the member of the class. My suggestion would be to call it `TryCacheNativeJoint` like in the other joints and use it in the same way.